components/faction/commons/faction.lua

```diff
@@ -128,7 +142,9 @@ function Faction.readMultiFaction(input, options)
 	end
 	---@cast input -nil
 
-	options = options or {}
+	if type(options) ~= 'table' then
```

Comment:
am on mobile comment is referring to this plus the next 2 lines

could make it 1 line
`options = type(options) == 'table' and options or {}`